Signal/src/ViewControllers/ConversationView/Cells/OWSMessageBubbleView.m

```diff
@@ -640,6 +648,10 @@ - (BOOL)canFooterOverlayMedia
 
 - (BOOL)hasBottomFooter
 {
+    if (self.shouldSuppressHeaderFooterAndRounding) {
```

Comment:
I thought we wanted the bubble to look more or less exactly the same in the message detail view as in the conversation.

I think it's a good call to avoid sharp corners in the detail view, but why don't we want the header and footer in message details view?

Reply:
I think we also always want to show the footer on the message detail screen.